packages/fxa-content-server/app/scripts/templates/sign_in_token_code.mustache

```diff
@@ -1,6 +1,6 @@
 <div id="main-content" class="card">
   <header>
-    <h1 id="fxa-signin-code-header" class="card-header">{{#unsafeTranslate}}Enter confirmation code<span class="card-subheader">for your Firefox account</span>{{/unsafeTranslate}}</h1>
+    <h1 id="fxa-signin-code-header" class="card-header">{{#unsafeTranslate}}Enter confirmation code <span class="card-subheader">for your Firefox account</span>{{/unsafeTranslate}}</h1>
```

Comment:
Oh oops, @pdehaan even asked me if this needed a space and I was like nah "for your Firefox account" goes on a new line... didn't think about the page title. I thought the "missing space" described in the ticket was _just_ around that "Code expired?" bit which was fixed in that other ticket so thanks for the due diligence here.

Reply:
@LZoog asking just in case you know offhand, how do screenreaders handle something like that?  Would they have interpreted that span as a pause/space?